lang/go/bindings/cne/system.go

```diff
@@ -123,6 +123,8 @@ func OpenWithFile(path string) (*System, error) {
 			if bytes, err := io.ReadAll(configFile); err == nil {
 				return open(jsonc.ToJSON(bytes))
 			}
+		} else {
+			return nil, err
```

Comment:
Wouldn't this just fall through to teh end of the function which returns nil, err?

Reply:
It appears the Go scoping rules have bit me. I assumed the declare of err at top of the function, that the os.Open() line would not create a new instance, but after testing it seems it does :-( Maybe something I am not understanding.

I reworked the code a bit to remove all of the := and that also seems to fix the problem.